composer.json

```
@@ -29,15 +29,16 @@
         }
     ],
     "require": {
-        "php": ">=7.1",
-        "sofa/hookable": "dev-7.0.1-erasys as 7.0.1",
-        "illuminate/database": ">=5.5"
+        "php": "^8.2",
+        "sofa/hookable": "8.0.1",
+        "illuminate/support": "^10.0",
+        "illuminate/database": "^10.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^9.0",
-        "squizlabs/php_codesniffer": "2.3.3",
-        "mockery/mockery": "^1.0",
-        "friendsofphp/php-cs-fixer": "^2.0"
+        "phpunit/phpunit": "^9.5",
+        "squizlabs/php_codesniffer": "^3.7",
+        "mockery/mockery": "^1.5",
+        "friendsofphp/php-cs-fixer": "^2.19"
```

Possible dependency conflict: In 'pr-php-code-standard', the package "friendsofphp/php-cs-fixer" version 3.18.0 is used. 

```suggestion
        "friendsofphp/php-cs-fixer": "3.18.0"
```